pkg/test/common.go

```diff
@@ -239,7 +235,7 @@ func CopyTestKeysAndCerts(destDir string) error {
 		"client.key", "server.cert", "server.csr", "server.key",
 	}
 
-	rootPath, err := GetProjectRootDir()
+	rootPath, err := image.GetProjectRootDir()
```

Comment:
This starts being confusing.
We're either using image, imageUtils, or dot imports.
It's too many variants for the same import. how about we use imageUtils or dot imports? Just 1 of these 2 variants.